### PR TITLE
File box

### PR DESCRIPTION
**Pull request by su-chang**
## I'm submitting a...

```
[ ] Bug Fix
[ ] Feature
[ ] Other (Refactoring, Added tests, Documentation, ...)
```

## Checklist

- [ ] Commit Messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern
  - A feature commit message is prefixed "feat:"
  - A bugfix commit message is prefixed "fix:"
- [ ] Tests for the changes have been added

## Description

_please describe the changes that you are making_

_for features, please describe how to use the new feature_

_please include a reference to an existing issue, if applicable_

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by su-chang** [comment]
Related issue: https://github.com/wechaty/wechaty/issues/1914

File: src/user/contact.spec.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ test('findAll()', async t => {
   const sandbox = sinon.createSandbox()
 
   const puppet = new PuppetMock()
-  const wechaty = new Wechaty({ puppet })
+  const wechaty = new Wechaty({ puppet: 'wechaty-puppet-mock' })
```
Comment (huan) [inline src/user/contact.spec.ts]: I believe that this will break the unit test.

Please revert.

File: src/user/message.spec.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ test('recalled()', async t => {
   const sandbox = sinon.createSandbox()
 
   const puppet = new PuppetMock()
-  const wechaty = new Wechaty({ puppet })
+  const wechaty = new Wechaty({ puppet: 'wechaty-puppet-mock' })
```
Comment (huan) [inline src/user/message.spec.ts]: `puppet` needs to be tested in the below assertions.

Please revert to make the unit test workable.

**Comment by su-chang** [comment]
In `wechaty-puppet-mock` will use FileBox from 'file-box', so we should import FileBox from `wechaty-puppet`?

**Comment by huan** [comment]
Yes. As discussed with @windmemory, we should import FileBox from `wechaty-puppet` in everywhere, including the `wechaty-puppet-mock`.

**Comment by su-chang** [comment]
So this PR will be test green after `wechaty-puppet-mock` PR merged? Right?

**Comment by huan** [comment]
No, it cannot.

Because the unit test needs to instantiate a puppet for testing purposes.

**Comment by su-chang** [comment]
But now it can not pass the CI test, could you give me some advices for this case?

**Comment by huan** [comment]
Cloud you please make sure you can pass the unit tests in your local environment by running `npm test`?

After that, I believe you can pass the CI.

**Comment by su-chang** [comment]
It can't, but the log of failed test show that:
```js
Error: Cannot find module 'file-box'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/suchang/Desktop/PROJECT/Test/testPadplus/wechaty/node_modules/wechaty-puppet-mock/src/puppet-mock.ts:21:1)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/suchang/Desktop/PROJECT/Test/testPadplus/wechaty/node_modules/wechaty-puppet-mock/src/index.ts:1:1)
```

So I think this error maybe due to the `FileBox` in `wechaty-puppet-mock` is from `file-box`?

**Comment by huan** [comment]
Yes. So I'd like to suggest that you can keep the dependencies of the file-box for now. I will remove it by later.

**Pull request merged**